### ISSUE TITLE
location example fails if feed does not exist

### ISSUE THREAD

**Issue by jerryneedell** [adafruit/io-client-python#57]
the example in api/location.py  failed if the requested feed does not exist:
it works if the feed is already there.
I created a feed named 'location' and the script worked fine.
I changed the requested feed to "newlocation" and it failed as below:

I am running this on a Raspberry Pi using adafruit-io v2.0.9

```
pi@gjnpi3p-1:~/projects/adafruitio/examples/api $ python3 location.py 
Traceback (most recent call last):
  File "location.py", line 23, in <module>
    location = aio.feeds('newlocation')
  File "/home/pi/.local/lib/python3.5/site-packages/Adafruit_IO/client.py", line 223, in feeds
    return Feed.from_dict(self._get(path))
  File "/home/pi/.local/lib/python3.5/site-packages/Adafruit_IO/client.py", line 88, in _get
    self._handle_error(response)
  File "/home/pi/.local/lib/python3.5/site-packages/Adafruit_IO/client.py", line 76, in _handle_error
    raise RequestError(response)
  File "/home/pi/.local/lib/python3.5/site-packages/Adafruit_IO/errors.py", line 40, in __init__
    error_message = self._parse_error(response)
  File "/home/pi/.local/lib/python3.5/site-packages/Adafruit_IO/errors.py", line 46, in _parse_error
    content = json.loads(response.content)
  File "/usr/lib/python3.5/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'bytes'
```

**Comment by jerryneedell** [comment]
I think the problem is that the returned exception is:
```
TypeError: the JSON object must be str, not 'bytes'
```
not  RequestError as tested for on line 24. If I just change to code to create the feed after any exception , it creates the feed correctly.

**Comment by jerryneedell** [comment]
ah -- the examples work on my Mac -- could this be a python 3.5 vs 3.6 issue?

**Comment by jerryneedell** [comment]
I installed python3.6 on my RPi and the examples now work.

**Comment by brentru** [comment]
This bug has been patched in master for 3.5.x and 3.6.x, closing...
